odoo-shippable/scripts/build-image.sh

```diff
@@ -51,7 +51,8 @@ DPKG_DEPENDS="postgresql-9.3 postgresql-contrib-9.3 postgresql-9.5 postgresql-co
 PIP_OPTS="--upgrade \
           --no-cache-dir"
 PIP_DEPENDS_EXTRA="line-profiler watchdog coveralls diff-highlight \
-                   pg-activity virtualenv nodeenv setuptools==33.1.1"
+                   pg-activity virtualenv nodeenv setuptools==33.1.1 \
+                   html2text==2016.9.19"
```

Comment:
@JesusZapata if this is an Odoo 10.0 dependency only, add it to the Odoo 10 image also please, because it will be needed there too.